samples/client/petstore/python/test_python2_and_3.sh

```diff
@@ -1,31 +0,0 @@
-#!/bin/bash
-
-REQUIREMENTS_FILE=dev-requirements.txt
-REQUIREMENTS_OUT=dev-requirements.txt.log
-SETUP_OUT=*.egg-info
-VENV=.venv
-DEACTIVE=false
-
-export LC_ALL=en_US.UTF-8
-export LANG=en_US.UTF-8
-
-### set virtualenv
-if [ -z "$VIRTUAL_ENV" ]; then
-		virtualenv $VENV --no-site-packages --always-copy
-		source $VENV/bin/activate
-    DEACTIVE=true
-fi
-
-### install dependencies
-pip install -r $REQUIREMENTS_FILE | tee -a $REQUIREMENTS_OUT
-
-### run tests
-tox || exit 1
-
-### static analysis of code
-flake8 --show-source petstore_api/
-
-### deactivate virtualenv
-#if [ $DEACTIVE == true ]; then
-#    deactivate
-#fi
```

Comment:
We need the files in samples/client/petstore/python/testfiles for testing, please add them back in

Reply:
Can you resolve this conversation?